<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QSpinBox" name="spinBoxValHigh">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="labelImgOrigin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QLabel" name="labelImgHSV">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSlider" name="horizontalSliderValHigh">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Hue low</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="horizontalSliderHueLow">
      <property name="maximum">
       <number>179</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hue high</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxHueLow">
      <property name="maximum">
       <number>179</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBoxHueHigh">
      <property name="maximum">
       <number>179</number>
      </property>
      <property name="value">
       <number>179</number>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QSlider" name="horizontalSliderHueHigh">
      <property name="maximum">
       <number>179</number>
      </property>
      <property name="value">
       <number>179</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sat low</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxSatLow">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="horizontalSliderSatLow">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Sat high</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Val low</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSlider" name="horizontalSliderSatHigh">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="spinBoxSatHigh">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxValLow">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Val High</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="horizontalSliderValLow">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Open image at</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEditImageAt">
      <property name="text">
       <string>C:\Users\yyyy\Qt\freelancer_jobs\build-hand_gestures_00-Desktop_Qt_5_6_3_MSVC2015_64bit-Release\24_letters_template\a1.jpg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxHueLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderHueLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderHueLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHueLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHueHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderHueHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderHueHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxHueHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSatLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSatLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSatLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSatLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSatHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSatHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSatHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSatHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxValLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderValLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderValLow</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxValLow</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxValHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderValHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderValHigh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxValHigh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
